### PR TITLE
JAVA: Add `GETBIT` command

### PR DESCRIPTION
**Pull request by GumpacG**

### CODE REVIEW

File: java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+     * @return The bit at offset of the string. Returns zero if the key is missing as it is treated as
+     *     an empty string. Returns zero if the positive offset exceeds the length of the string as it
+     *     is assumed to be padded zeroes.
+     * @example
+     *     <pre>{@code
+     * Long payload = client.getbit("myKey1", 1).get();
+     * assert payload == 1L; // The second bit for string stored at "myKey1" is set to 1.
+     * }</pre>
+     */
+    CompletableFuture<Long> getbit(String key, long offset);
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java:28]: TBD maybe return bool?

File: java/integTest/src/test/java/glide/SharedCommandTests.java [hunk 2]
```diff
@@ -3290,4 +3290,27 @@ public void geoadd_invalid_args(BaseClient client) {
...
+        String missingKey = "missing";
+        String value = "foobar";
+
+        assertEquals(OK, client.set(key1, value).get());
+        assertEquals(1, client.sadd(key2, new String[] {value}).get());
+        assertEquals(1, client.getbit(key1, 1).get());
+        assertEquals(0, client.getbit(key1, 1000).get());
+        assertEquals(0, client.getbit(missingKey, 1).get());
+
+        ExecutionException executionException =
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/SharedCommandTests.java:3309]: Please add comments to error cases.

**Review by Yury-Fridlyand — APPROVED** [review]

File: java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java (hunk 1 above)
Comment (GumpacG) [reply java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java:28]: That would differ from what Redis returns though

Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java:28]: Yes, but we have 0/1 conversion to bool in some commands.
It is also applicable here, especially if we reword the doc a bit
> Checks whether bit is set ...

Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java:28]: Not mandatory though, just an idea

**Pull request merged**